schemas/process.yml

```diff
@@ -61,9 +61,21 @@
         short_override: An array of previous executions for the process, including the initial fork. Only executable and args are set.
         normalize:
           - array
+      - at: process
+        as: responsible
+        short_override: Responsible process in macOS tracks the originating process of an app, key for understanding permissions and hierarchy.
+        beta: This field is beta and subject to change.
 
   fields:
 
+    - name: endpoint_security_client
+      level: extended
+      type: boolean
+      short: Indicates whether this process executable is an Endpoint Security client.
```

Comment:
This might have been a question to previous round but what is endpoint security client? Any security client on mac endpoint? Capitalisation in short makes me think this is a name of a product. But description has again lower case

Reply:
Hey!

It defines if the the instigator of the event is a Endpoint security client or not.

https://developer.apple.com/documentation/endpointsecurity/es_process_t/3228978-is_es_client